hudi-hive/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -75,6 +75,7 @@
 public class HoodieHiveClient {
 
   private static final String HOODIE_LAST_COMMIT_TIME_SYNC = "last_commit_time_sync";
+  private static final String HOODIE_FIRST_COMMIT_TIME = "0";
```

Comment:
Instead of special casing like this, we can change the method you call to `findPartitionsAfter(HoodieTimeline)` and pass  `activeTimeline` in `if` and `activeTimeline.findInstantsAfter(lastCommitTimeSynced, Integer.MAX_VALUE)` in the `else`

Reply:
If so, we need two methods, code will be redundant.
 